virt/virt-2-5-release-notes.adoc

```diff
@@ -129,6 +129,8 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 
 * When a node fails in user-provisioned installations of {product-title} on bare metal deployments, the virtual machine does not automatically restart on another node. Automatic restart is supported only for installer-provisioned installations that have machine health checks enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886437[*BZ#1886437*])
 
+* It is not possible to attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, you can use a secondary network interface connected to your host or switch to OpenShift SDN. link:https://bugzilla.redhat.com/show_bug.cgi?id=1893160[(*BZ#1893160*)]
```

Comment:
I think we should clearly state that this is an issue only in case OVN Kubernetes is used as the default CNI provider.

Reply:
@phoracek I have revised the text, can you please re-review?